packages/atoms/src/components/kiwi-modal/kiwi-modal.tsx

```diff
@@ -175,52 +101,22 @@ export class KiwiModal {
                   >
                     ×
                   </button>
+
                   <h4 class="modal-title">
-                    <slot name="modal-title"></slot>
+                    <slot name="kiwi-modal-title"></slot>
                   </h4>
                 </div>
               )}
+
               <div class="modal-body">
-                <slot name="modal-body"></slot>
+                <slot name="kiwi-modal-body"></slot>
               </div>
-              {some([
-                this.cancelText,
-                this.previousText,
-                this.nextText,
-                this.okText,
-              ]) && (
-                <div class="modal-footer">
-                  {this.previousText && (
-                    <button
-                      class="btn btn-link pull-left"
-                      onClick={this.handlePrevious}
-                    >
-                      {this.previousText}
-                    </button>
-                  )}
-                  {this.cancelText && (
-                    <button class="btn btn-link" onClick={this.handleClose}>
-                      {this.cancelText}
-                    </button>
-                  )}
-                  {this.nextText && (
-                    <button class="btn btn-primary" onClick={this.handleNext}>
-                      {this.nextText}
-                    </button>
-                  )}
-                  {this.okText && (
-                    <button
-                      class="btn btn-primary"
-                      onClick={this.handleConfirmation}
-                    >
-                      {this.okText}
-                    </button>
-                  )}
-                </div>
-              )}
+
+              <slot name="kiwi-modal-footer"></slot>
```

Comment:
Could you please document the slots `kiwi-modal-title`, `kiwi-modal-body` and `kiwi-modal-footer`.
Especially that `kiwi-modal-footer` has to include the markup
```html
<div class="modal-footer"></div>
```

Reply:
Or instead of requiring the footer having to include this markup, you could add a `@Prop()` like `withFooter` and change it to
```jsx
{this.withFooter && <div class="modal-footer">
  <slot name="kiwi-modal-footer"></slot>
</div>}
```

Reply:
> Especially that `kiwi-modal-footer` has to include the markup
> 
> ```
> <div class="modal-footer"></div>
> ```

The fact that there is a certain markup required should be hidden from the user. That's why there is a `<kiwi-modal-footer>` component abstracting this away. We tried to make this more explicit in the slot documentation.